app/models/orphan.rb

```diff
@@ -134,11 +136,15 @@ def sponsored_siblings_does_not_exceed_siblings_count
   end
 
   def default_sponsorship_status_unsponsored
-    self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    if new_record?
+      self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    end
   end
 
   def default_orphan_status_active
-     self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    if new_record?
+      self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    end
```

Comment:
There is also `default_priority_to_normal` on L158. What are your thoughts on combining all these initializers into a single method?